synapseclient/models/schema_organization.py

```diff
@@ -0,0 +1,1344 @@
+            ValueError: This JSONSchema doesn't have an organization name
+            ValueError: This JSONSchema doesn't exist in its organization
+
+        Returns:
+            Itself
+
+        Example: Get an existing JSONSchema
+            &nbsp;
+
+            ```python
+            from synapseclient.models import JSONSchema
+            from synapseclient import Synapse
+            import asyncio
+
+            async def get_schema():
+
+                syn = Synapse()
+                syn.login()
+
+                schema = JSONSchema(organization_name="dpetest", name="test.schematic.Biospecimen")
+                await schema.get_async()
+                return schema
+
+            schema = asyncio.run(get_schema())
+            print(schema.uri)
+            ```
+        """
+        if not self.name:
+            raise ValueError("JSONSchema must have a name")
+        if not self.organization_name:
+            raise ValueError("JSONSchema must have a organization_name")
+
+        # Check that the org exists,
+        #  if it doesn't list_json_schemas will unhelpfully return an empty generator.
+        org = SchemaOrganization(self.organization_name)
+        await org.get_async(synapse_client=synapse_client)
+
+        org_schemas = list_json_schemas(
+            self.organization_name, synapse_client=synapse_client
+        )
+        async for schema in org_schemas:
+            if schema["schemaName"] == self.name:
+                self.fill_from_dict(schema)
+                return self
+        raise ValueError(
+            (
+                f"Organization: '{self.organization_name}' does not contain a schema with "
+                f"name: '{self.name}'"
+            )
+        )
+
+    async def store_async(
+        self,
+        schema_body: dict[str, Any],
+        version: Optional[str] = None,
+        dry_run: bool = False,
+        synapse_client: Optional["Synapse"] = None,
+    ) -> "JSONSchema":
+        """
+        Stores this JSONSchema in Synapse
+
+        Arguments:
+            schema_body: The body of the JSONSchema to store
+            version: The version of the JSONSchema body to store
+            dry_run: Whether or not to do a dry-run
+            synapse_client: If not passed in and caching was not disabled by
+                `Synapse.allow_client_caching(False)` this will use the last created
+                instance from the Synapse class constructor
+
+        Returns:
+            Itself
+
+        Example: Store a JSON Schema in Synapse
+            &nbsp;
+
+            ```python
+            from synapseclient.models import JSONSchema
+            from synapseclient import Synapse
+            import asyncio
+
+            async def store_schema():
+
+                syn = Synapse()
+                syn.login()
+
+                schema = JSONSchema(organization_name="my.org", name="test.schema")
+                await schema.store_async(schema_body = {})
```

Comment:
Nit: I think it is a bit confusing to have an empty schema body in the example. Could you provide a simple schema as an example?